p2p/simulations/adapters/types.go

```diff
@@ -211,6 +228,7 @@ func RandomNodeConfig() *NodeConfig {
 		Name:            fmt.Sprintf("node_%s", enodId.String()),
 		Port:            port,
 		EnableMsgEvents: true,
+		LogVerbosity:    log.LvlUndefined,
```

Comment:
This is the only place where LvlUndefined is used. Can't we just use 0 here?

Reply:
0 is the critial. So I added this stuff.

Reply:
Or we can just pass the `log.LvlInfo` by default.